src/installation/index.md

```diff
@@ -1,160 +1,198 @@
-- [`ldproxy`] crate:  Simple tool to forward linker arguments given to [`ldproxy`] to the actual linker executable. The crate can be found in the [esp-rs/embuild] repository. To install it, use the following command:
-  - `cargo install ldproxy`
+After running `espup install`:
 
+- `no_std` (bare-metal) applications should work out of the box
+- `std` applications require additional software covered in [`std` Development Requirements][rust-esp-book-std-requirements]
+
+### What espup Installs
+
+To enable support for Xtensa targets, `espup` installs the following tools:
+
+- Espressif Rust fork with support for `Xtensa` targets
+- `nightly` toolchain with support for `RISC-V` targets
+- `LLVM` [fork][llvm-github-fork] with support for `Xtensa` targets
+- [GCC toolchain][gcc-toolchain-github-fork] as it is used as linker
+
+The forked compiler can coexist with the standard Rust compiler, allowing both to be installed on your system. The forked compiler is invoked when using any of the available [overriding methods][rustup-overrides].
+
+> **Note**: We are making efforts to upstream our forks
+> 1. Changes in `LLVM` fork. Already in progress, see the status in this [tracking issue][llvm-github-fork-upstream issue].
+> 2. Rust compiler forks. If `LLVM` changes are accepted, we will proceed with the Rust compiler changes.
+
+
+[llvm-github-fork]: https://github.com/espressif/llvm-project
+[gcc-toolchain-github-fork]: https://github.com/espressif/crosstool-NG/
+[rustup-overrides]: https://rust-lang.github.io/rustup/overrides.html
+[llvm-github-fork-upstream issue]: https://github.com/espressif/llvm-project/issues/4
+
+
+### Other installation methods for Xtensa targets
+
+- Using [esp-rs/rust-build] installation scripts. This was the recommended way in the past, but now the installation scripts are feature frozen, and all new features will only be included in `espup`. See the repository README for instructions.
+- Building the Rust compiler with `Xtensa` support from source. This process is computationally expensive and can take one or more hours to complete depending on your system. It is not recommended unless there is a major reason to go for this approach. Here is the repository to build it from source: [esp-rs/rust repository].
+
+[esp-rs/rust-build]: https://github.com/esp-rs/rust-build#download-installer-in-bash
+[esp-rs/rust repository]: https://github.com/esp-rs/rust
+
+
+## `std` Development Requirements
+
+Regardless of the target architecture, make sure you have the following required tools installed to build [`std`][rust-esp-book-overview-std] applications:
+
+- [`python`][python-website-download]: Required by ESP-IDF
+- [`git`][git-website-download]: Required by ESP-IDF
+- [`ldproxy`][embuild-github-ldproxy] crate: Simple tool to pass the `ldproxy` linker arguments to the actual linker executable. Install it  by running:
+
```

Comment:
We are still missing the codeblock with the installation command:
```
cargo install ldproxy
```

Reply:
Yeah, thank!